packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,17 +2,44 @@
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-panel-content-space: Specifies the padding of the component's content.
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-shadow: Specifies the shadow of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
```

Comment:
I think this can be simplified to `--calcite-panel-text-color` which will be the default text color for the component. Because color.text.2 is the default text color for components so having this be set on the main container and allow it to cascade through the component by default will align better with our design patterns.

Reply:
I don't understand - these aren't for slotted content, these are rendered based on the presence of a property, `heading`, or `description`, which each have a different default color.

Reply:
If a user wants to make their `heading` red, and their description `blue` - using global text color variables would affect all child components (and can already be done)...